r-markdown.md

```diff
@@ -0,0 +1,48 @@
+---
+title:  r  
+description: r  Cheat Sheet gives you a quick reference of basic r markdown language. 
+First appeared:	August 1993
+Stable release:	4.2.1 / 23 June 2022
+---
+  
+## What is ```r``` ?
+Introduction to ```r```.<br>
+```r```  is a language and environment for statistical computing and graphics. It is a GNU project which is similar to the S language and environment which was developed at Bell Laboratories (formerly AT&T, now Lucent Technologies) by John Chambers and colleagues. 
+
+```r``` provides a wide variety of statistical (linear and nonlinear modelling, classical statistical tests, time-series analysis, classification, clustering, …) and graphical techniques, and is highly extensible. The S language is often the vehicle of choice for research in statistical methodology, and ```r``` provides an Open Source route to participation in that activity.
```

Comment:
Can you reduce the description and increase code samples, please? This is more of a cheat sheet than like a Tutorial

Reply:
Issue resolved_